frontend/src/AdjustMetricUnit.res

```diff
@@ -1,50 +1,80 @@
-let isSize = (x) => Js.Re.exec_(%re("/(gb|mb|kb|bytes)\w*/i"), x)
-  ->Belt.Option.isSome
+let sizeUnits = ["bytes", "kb", "mb", "gb", "tb", "pb", "eb", "zb", "yb"]
+// FIXME: Should include other units too?!
```

Comment:
Yes it should; I think it might be nice to abstract as a module to convert all the units :) but that can be another PR